account-kit/react/src/components/auth/card/error/connection-error.tsx

```diff
@@ -20,8 +21,29 @@ export const ConnectionError = ({
   EOAConnector,
   handleTryAgain,
   handleUseAnotherMethod,
+  customErrorMessage,
 }: ConnectionErrorProps) => {
+  const signer = useSigner();
+
+  useEffect(() => {
+    // Terminate any inflight authentication on Error...
+    if (
+      connectionType === "wallet" &&
+      EOAConnector === EOAWallets.WALLET_CONNECT
+    ) {
+      clearWalletConnectStore();
+    }
+
+    if (signer) {
+      signer.disconnect();
+    }
```

Comment:
we should use the `import { disconnect } from "@account-kit/core";` or does this cause some weird stuff to happen?